pkgs/by-name/re/reminders/package.nix

```diff
@@ -0,0 +1,88 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+
+  meson,
+  ninja,
+  pkg-config,
+  appstream,
+  glib,
+  gtk4,
+  desktop-file-utils,
+  gobject-introspection,
+  wrapGAppsHook4,
+
+  libadwaita,
+  gsound,
+  webkitgtk_6_0,
+  libsecret,
+}:
+python3.pkgs.buildPythonApplication {
+  pname = "reminders";
+  # 4.0 release is pre-rebranding and still called "Remembrance"
+  version = "4.0-unstable-2023-05-03";
+  pyproject = false;
+
+  src = fetchFromGitHub {
+    owner = "remindersdevs";
+    repo = "Reminders";
+    rev = "f649ea653b43d3ff0ef331729eb043fbb912f6f7";
+    hash = "sha256-dhtspgHM+HWqDSNdF4O/NOyDGcL7aADsm0yT5MKjw3k=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    appstream # `appstreamcli`
+    glib # `glib-compile-schemas`
+    gtk4 # `gtk-update-icon-cache`
```

Comment:
Could you set strictDeps = true to ensure these are really only used during the build, not linked into the result?